src/Session.php

```diff
@@ -50,7 +51,12 @@ public function __get(string $name)
             throw new RuntimeException('Session not initialized');
         }
 
-        return $this->contents[$name] ?? null;
+        if (!isset($this->contents[$name])) {
```

Comment:
On further review, this is the wrong approach. Don't trigger an error, just restore the line above and remove the null coalesce. PHP will handle triggering the error since $name does not exist as an element in $this->contents.